Makefile

```diff
@@ -21,51 +21,9 @@ endif
 DOCKER_BUILD := docker build \
 	--build-arg GO_VERSION=${GO_VERSION}
 
-# currently disabled linters:
-#   gofmt               # enable after switch to go1.11
-#   goimpor             # enable after switch to go1.11
-#   unparam             # enable later
-#   gosec               # enable later
-#   vetshadow           # enable later
-#   gochecknoinits      # enable later
-#   gochecknoglobals    # enable later
-#   dupl                # needs tuning
-#   gocyclo             # needs tuning
-#   lll                 # long lines
-#   gotype, gotypex     # already done by compiling
-#   safesql             # no sql
-#   interfacer          # not that useful
 lint: check-copyrights
 	@echo "Running ${@}"
-	@gometalinter \
-	--deadline=10m \
-	--concurrency=1 \
-	--enable-all \
-	--enable=golint \
-	--enable=errcheck \
-	--enable=unconvert \
-	--enable=structcheck \
-	--enable=misspell \
-	--disable=goimports \
-	--enable=ineffassign \
-	--disable=gofmt \
-	--enable=nakedret \
-	--enable=megacheck \
-	--disable=unparam \
-	--disable=gosec \
-	--disable=vetshadow \
-	--disable=gochecknoinits \
-	--disable=gochecknoglobals \
-	--disable=dupl \
-	--disable=gocyclo \
-	--disable=lll \
-	--disable=gotype --disable=gotypex \
-	--disable=safesql \
-	--disable=interfacer \
-	--skip=examples \
-	--exclude=".*\.pb\.go" \
-	--exclude=".*\.dbx\.go" \
-	./...
+	@golangci-lint run ./...
```

Comment:
so much cleaner!

Reply:
The readme suggests that `./...` is even too much: https://github.com/golangci/golangci-lint#quick-start

Reply:
force of habit, will fix